src/C-interface/bml_allocate.c

```diff
@@ -21,12 +21,12 @@
  *
  * \ingroup allocate_group_C
  *
- * \param A Matrix
- * \return >0 if allocated, else -1
+ * \param A[in,out] Matrix
+ * \return \f$ > 0 \f$ if allocated, else -1
```

Comment:
What is \f$ supposed to mean here?

Reply:
That's doxygen's way to include a LaTeX expressions. I figured it would look nicer to typeset that properly :smile:

Reply:
http://www.doxygen.nl/manual/formulas.html